deeplinkdispatch-base/src/main/java/com/airbnb/deeplinkdispatch/base/MatchIndex.java

```diff
@@ -12,41 +13,28 @@
 
 /**
  * This is a wrapper class around the byte array match index.
+ * <p>Byte array format is:</p>
+ * <table border="1">
```

Comment:
Does the table render nicer in IntelliJ? -- It's a lot harder to read raw.

Reply:
no, it's terrible everywhere. It will render in an actual javadoc. I think the old format (ascii art) is unmaintainable though.